tests/cluster_wide_install/cluster_wide_install_suite_test.go

```diff
@@ -60,7 +61,7 @@ var _ = Describe(testName, func() {
 			//step := "setting up cass-operator resources via helm chart"
 
 			var overrides = map[string]string{
-				"image":              ginkgo_util.OperatorImage,
+				"image":              cfgutil.GetOperatorImage(),
```

Comment:
Err, this one was my fault. I didn't see the `OperatorImage` references in these tests.